data/scripts/talkactions/gm/getlook.lua

```diff
@@ -4,11 +4,10 @@ function getlook.onSay(player, words, param)
 	-- create log
 	logCommand(player, words, param)
 
-	local lookt = Creature(player):getOutfit()
+	local lookt = player:getOutfit()
 	player:sendTextMessage(MESSAGE_HOTKEY_PRESSED, "<look type=\"" .. lookt.lookType .. "\" head=\"" .. lookt.lookHead .. "\" body=\"" .. lookt.lookBody .. "\" legs=\"" .. lookt.lookLegs .. "\" feet=\"" .. lookt.lookFeet .. "\" addons=\"" .. lookt.lookAddons .. "\" mount=\"" .. lookt.lookMount .. "\" />")
 	return true
 end
 
-getlook:separator(" ")
```

Comment:
Why are you removing separator? We need to use `/getlook <player_name>`

Reply:
The file does not use parameters! If the actual use of this command is to use a character's name, it is wrong. It was getting the player itself in "Creature(player)" not "Creature(param)"

Reply:
> The file does not use parameters! If the actual use of this command is to use a character's name, it is wrong. It was getting the player itself in "Creature(player)" not "Creature(param)"

Then you can fix.. This command is for gm get look of another player, I think.

Reply:
done!